cmd/manager/main.go

```diff
@@ -109,6 +111,7 @@ func main() {
 		Namespace:          namespace,
 		MapperProvider:     restmapper.NewDynamicRESTMapper,
 		MetricsBindAddress: fmt.Sprintf("%s:%d", metricsHost, metricsPort),
+		SyncPeriod:         &syncperiod,
```

Comment:
I am not familiar with this, what does it do?

Reply:
I'm pretty sure we've talked about this before, no? :thinking: It's a time period at which the operator will force a reconcile, to make sure everything is correct (in case for some reason an operator missed a trigger, or one wasn't sent or something).